Model/Api.php

```diff
@@ -485,6 +485,16 @@ public function containerValidateServiceVersion($version)
         return $result;
     }
 
+    /**
+     * @return bool|mixed
+     * @throws LocalizedException
+     */
+    public function getDataCenters()
+    {
+        $url = $this->config->getApiEndpoint() . 'datacenters';
+        return $this->_fetch($url, 'GET');
+    }
```

Comment:
@vvuksan and I are on the same page. We don't want to call /datacenters API in this way but rather want to have a static JSON file in our repo for the time being. This is not just about stability but also ensure we don't harm our own platform (API backend) by this change since we have fairly many plugin users out there.

Reply:
HI @smaeda-ks ,

I agree with these redundant functions. I've changed the code and did what you have requested. 
However, I still think that pinging `/datacenter` is not a big deal and its much better solution then static .json file.

Reply:
Hmm, I think you're right. But then our minimum requirement is that we must catch errors from the `/datacenter` API endpoint (e.g., 503) and actually shows an error message to the logged-in user to tell "hey, something went wrong". And if there's an error at this stage we don't want to allow users to submit the form at all. So basically, users need to refresh the page or retry this flow until they can successfully fetch the shielding data from our API.

Reply:
@smaeda-ks 
Totally understands. I've removed the pinging. Thanks for collaboration